frontend/src/app/modules/shared/modules/notification/modal/accept/accept-notification-modal.component.ts

```diff
@@ -34,7 +35,7 @@ import { Observable } from 'rxjs';
 export class AcceptNotificationModalComponent {
   @ViewChild('Modal') modal: TemplateRef<unknown>;
   @Input() acceptCall: (id: string, reason: string) => Observable<void>;
-  @Input() translationContext: 'commonInvestigation' | 'commonAlert';
+  @Input() translationContext: string;
```

Comment:
Base on changes I proposed below, we can use here TranslationContext type, and adjust passing this value in templates. That will be more consistent and clear how to use translationContext value.

```suggestion
  @Input() translationContext: TranslationContext;
```